test/runnable/test19441.d

```diff
@@ -0,0 +1,23 @@
+struct S1
```

Comment:
TEST_OUTPUT is missing here.

Reply:
There is no output. This is valid D code.

Reply:
That's not an argument against having an empty section to ensure that no  depreciations are  triggered. We going to require the test_output for every test soon.

Reply:
ok. done